lib/pharos/scripts/configure-essentials.sh

```diff
@@ -7,3 +7,8 @@ if ! dpkg -l apt-transport-https software-properties-common > /dev/null; then
     apt-get update -y
     apt-get install -y apt-transport-https software-properties-common
 fi
+
+cat <<EOF >/etc/apt/apt.conf.d/20auto-upgrades
+APT::Periodic::Update-Package-Lists "1";
```

Comment:
Note that this is still required for `unattended-upgrade` to work, I don't think it will do the equivalent of `apt-get update` by itself.